tests/test_efel_settings.py

```diff
@@ -28,21 +29,21 @@ def setUp(self):
 
     def test_efel_threshold(self):
         
-        self.cell.recordings[0].efeatures = {}
+        self.cell.recordings["IDRest"][0].efeatures = {}
 
         self.cell.extract_efeatures(
             protocol_name="IDRest",
             efeatures=["Spikecount", "AP1_amp"],
             efel_settings={'Threshold': 40.}
         )
 
-        recording = self.cell.recordings[0]
+        recording = self.cell.recordings["IDRest"][0]
         self.assertEqual(recording.efeatures["Spikecount"], 0.)
-        self.assertLess(abs(recording.efeatures["AP1_amp"] - 66.68), 0.01)
```

Comment:
this code was not getting executed for a long time the other test was shadowing it